src/ng/directive/ngOptions.js

```diff
@@ -147,37 +147,39 @@ var ngOptionsMinErr = minErr('ngOptions');
         <div ng-controller="ExampleController">
           <ul>
             <li ng-repeat="color in colors">
-              Name: <input ng-model="color.name">
-              <input type="checkbox" ng-model="color.notAnOption"> Disabled?
-              [<a href ng-click="colors.splice($index, 1)">X</a>]
+              <label>Name: <input ng-model="color.name"></label>
+              <label><input type="checkbox" ng-model="color.notAnOption"> Disabled?</label>
+              [<button ng-click="colors.splice($index, 1)" aria-label="remove">X</button>]
```

Comment:
Maybe a separate PR. Also, in a similar change earlier in these changes the brackets were dropped. That's inconsistent with this change. Not sure which you prefer.